easybuild/easyconfigs/d/datamash/datamash-1.3-foss-2018a.eb

```diff
@@ -0,0 +1,24 @@
+# Author: Piotr Malicki
+
+easyblock = 'ConfigureMake'
+
+name = 'Datamash'
```

Comment:
@PiotrCM The right software name seems to be `datamash` (cfr. https://www.gnu.org/software/datamash/), please change?